src/components/item/Item.js

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+import FavButton from "../FavButton/FavButton";
+
+export default function Item({
+  title,
+  text,
+  date,
+  id,
+  isFavourite,
+  onToggleFavourite,
+}) {
+  return (
+    <StyledEntry>
+      <p className="date">{date}</p>
+      <p className="title">
+        {title}
+        <FavButton
+          id={id}
+          isFavourite={isFavourite}
+          onToggleFavourite={onToggleFavourite}
+        />
+      </p>
+
+      <p className="text">{text}</p>
+    </StyledEntry>
+  );
+}
+
+const StyledEntry = styled.li`
+  padding: 0.2rem;
+  margin: 1rem 0;
+  .title {
+    font-weight: bold;
+    margin: 0;
+    margin-right: 1rem;
+    display: flex;
+    justify-content: space-between;
+  }
+  .date {
+    font-size: 0.8rem;
+    margin: 0.2rem 0;
+  }
+  .text {
+    margin-top: 0.5rem;
+  }
+`;
```

Comment:
Cool that this works!
I haven't seen someone use styled components with classnames, but in this case it makes sense :)

Reply:
I wanted to try out different methods. And because I already had the classnames from before Styled Components It tried out of laziness :)